src/runtime_src/core/pcie/tools/xbmgmt/xmc.cpp

```diff
@@ -38,10 +38,9 @@ XMC_Flasher::XMC_Flasher(std::shared_ptr<pcidev::pci_device> dev)
     bool is_mfg = false;
     mDev->sysfs_get<bool>("", "mfg", err, is_mfg, false);
     if (!is_mfg) {
-        if (mDev->get_sysfs_path("xmc", "").empty()) {
-            mProbingErrMsg << "Failed to detect XMC"; 
+         /* Don't print any error here because Recovery BLP doesn't have xmc */
+         if (!hasXMC())
```

Comment:
> will this bring http://jira.xilinx.com/browse/CR-1053826 back?

If the roto casue is "stop getting boardInfo when xmc is not there", then yes, this won't bring the CR back.
Because we added code to stop when !hasXMC() in Flasher::getBoardInfo

Reply:
seems the above comment is gone in the latest code, please refresh and recheck.